packages/addons/addon-depends/hidapi/package.mk

```diff
@@ -0,0 +1,12 @@
+# SPDX-License-Identifier: GPL-2.0-or-later
+# Copyright (C) 2023-present Team LibreELEC (https://libreelec.tv)
+
+PKG_NAME="hidapi"
+PKG_VERSION="0.14.0"
+PKG_SHA256="a5714234abe6e1f53647dd8cba7d69f65f71c558b7896ed218864ffcf405bcbd"
+PKG_LICENSE="HIDAPI-orig"
```

Comment:
The licence is clusterfu
```
1. The GNU General Public License, version 3.0, in LICENSE-gpl3.txt
2. A BSD-Style License, in LICENSE-bsd.txt.
3. The more liberal original HIDAPI license. LICENSE-orig.txt
```
https://github.com/libusb/hidapi/blob/master/LICENSE.txt so we would "need" 3 licences here -.-
not sure how we handle that

Reply:
This just means it's up to us to choose the license we use (from the three available) so I picked the original (most liberal) one. No need to do more.